ci/gather_diagnostics_data.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -eu -o pipefail
+
+echo "Services list: $1"
+
+IFS=',' read -ra SERVICES <<< "$1"
+
+if [ -z "${SERVICES}" ]; then
+  echo SERVICES is null or empty.
+  exit 1
+fi
+
+chmod +x ./python/create_service_diagnostics_bundle.sh
+export TTY_OPTS="-t"
+for service in "${SERVICES[@]}"; do
+    echo "Starting service diagnostics script for ${service}"
+    ./python/create_service_diagnostics_bundle.sh \
+        --package-name="${service}" \
+        --service-name="${service}" \
+        --yes || true &
+done
+
+wait
```

Comment:
Why are we ignoring the return code of the scripts?

Reply:
how would you propose handle it?

Reply:
There are ways to do it in plain shell, but not pretty. There's also the super-handy GNU `parallel` utility, but I don't know whether we want/can add it as a dependency. Perhaps it would be best to do it in Python like you did for `ci/install_data_services.py`.

Reply:
Yes, it all sounds reasonable. But I'd prefer don't change current logic, until you have an example when the code can break expected behavior. 

Logs output from forked processes are going to stdout and error messages monitored by TC (I guess).

Reply:
We should really be checking the exit codes, rather than applying heuristics to stderr to determine whether something succeeded or not.